tables/vpc_flow_log/vpc_flow_log_extractor.go

```diff
@@ -32,26 +40,21 @@ func (c *VPCFlowLogExtractor) Extract(_ context.Context, a any) ([]any, error) {
 		return nil, fmt.Errorf("expected byte[], got %T", a)
 	}
 
-	mappedData, err := ConvertToMapSlice(string(jsonBytes))
+	mappedData, err := ConvertToMapSlice(c, string(jsonBytes))
 	if err != nil {
 		return nil, fmt.Errorf("error in mapping the results in the form of []map[string]string")
 	}
 
 	var res = make([]any, len(mappedData))
 	for i, record := range mappedData {
 
-		flowLog := &VpcFlowLog{}
-		err := flowLog.MapValues(record)
-		if err != nil {
-			return nil, fmt.Errorf("error in extracting the log: %v", err)
-		}
-		res[i] = flowLog
+		res[i] = record
 	}
 	return res, nil
 }
 
 // ConvertToMapSlice converts space-separated string data into a slice of map[string]string
-func ConvertToMapSlice(data string) ([]map[string]string, error) {
+func ConvertToMapSlice(c *VPCFlowLogExtractor, data string) ([]map[string]string, error) {
```

Comment:
does this function need to be exported?

Can we give it a better name?

Reply:
No, it doesn't need to be exported. Updated the name to use `toMapSlice` instead.